File: JS-Applications-November-2016/09. ReactJS/Lab/Component-Example/src/Person.js
import React, { Component } from 'react';
import './Person.css';

class Person extends Component {
    render() {
        return (
            <div>
                <h1>{this.props.name}'s Page</h1>
                <p>Tel. {this.props.phone}</p>
            </div>
        )
    }
}

export default Person;

File: Databases-Advanced-Entity-Framework-October-2017/02. Defining-Classes-Exercises/Exercises/Creating-Constructors/Person.cs
public class Person
{
    public string Name;
    public int Age;

    public Person(string name, int age)
    {
        this.Name = name;
        this.Age = age;
    }

    public Person(int age)
        : this("No name", age)
    {
    }

    public Person()
        : this("No name", 1)
    {
    }
}